.github/ISSUE_TEMPLATE/bug_report_template.yaml

```diff
@@ -0,0 +1,83 @@
+name: Bug Report
+description: Report a bug encountered while operating Tetragon
+labels: ["kind/community-report", "kind/bug", "needs/triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! If you have usage questions, please try the [slack channel](https://cilium.herokuapp.com/) and see the [FAQ](https://github.com/cilium/tetragon#faq) first.
```

Comment:
Can you link to the Join the Slack using this URL: https://slack.cilium.io/
Can you link to the specific Tetragon Slack channel? https://cilium.slack.com/archives/C03EV7KJPJ9

Like "Join the Cilium Slack and ask questions in the Tetragon channel"